src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dao/trust/TrustCorporateDao.java

```diff
@@ -42,6 +42,12 @@ public class TrustCorporateDao {
     @Field("is_on_register_in_country_formed_in")
     private Boolean onRegisterInCountryFormedIn;
 
+    @Field("is_corporate_body_still_involved_in_overseas_entity")
```

Comment:
Relates to requirements, but is this story all about checking the ceasing of a legal entity (= corporate body) with a trust? If yes, should this field be named `is_corporate_body_still_involved_in_trust`?

Reply:
Good point will change that.